Assets/Scripts/Game/InputManager.cs

```diff
@@ -47,6 +47,8 @@ public class InputManager : MonoBehaviour
         const float deadZone = 0.05f;
         const float inputWaitTotal = 0.0833f;
         const float moveAccelerationConst = 9.8f;
+        const float controllerCursorHorizontalSpeed = 900.0f;
```

Comment:
Any reason these were changed from 300.0f to 900.0f?

Reply:
When switching over to the Update function, the cursor speed became slower. I think it's because I read that OnGUI runs more times in a frame than Update, which may be why switching to Update decreased the speed.

Reply:
Yeah okay, that makes sense